src/plot_settings.py

```diff
@@ -157,4 +157,5 @@ def on_close(self, *_args):
             ui.reload_item_menu(self.props.application)
         else:
             self.props.application.canvas.load_limits()
+            self.props.application.canvas.toolbar.push_current()
```

Comment:
Similar question here as before, what is the reasoning that a push_current is needed now here?